docs/sql-ref-syntax-aux-show-table.md

```diff
@@ -18,5 +18,147 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+SHOW TABLE EXTENDED will show information for all tables matching the given regular expression.
+Output includes basic table information and file system information like `Last Access`, 
+`Created By`, `Type`, `Provider`, `Table Properties`, `Location`, `Serde Library`, `InputFormat`, 
+`OutputFormat`, `Storage Properties`, `Partition Provider`, `Partition Columns` and `Schema`.
+
+Users cannot use regular expression for table name if a partition specification is present and  it will output 
```

Comment:
"for _the_ table name"
"is present. It will output"

Reply:
Done